variables.tf

```diff
@@ -1,20 +1,16 @@
 variable "app_name" {
```

Comment:
Do you think we should start alphabetizing our variables?

Reply:
It might just be my preference, but I liked grouping variables by function and sorting by importance. But if we want we can alphabetize like the providers have them in their docs. Thoughts?